staff/daniela/Precurso/Calculator-2/index.js

```diff
@@ -0,0 +1,69 @@
+
+
+    const sendButton = document.querySelector(".sendButton");
+    const firstNumber = document.querySelector("#firstParameter");
+    const secondNumber = document.querySelector("#secondParameter");
+    const mathResults = document.querySelector(".instructions");
+    //console.log(sendButton);
```

Comment:
@codingwithdani commented code here do nothing, you can erase it

Reply:
done!